cob_bringup/robots/cob3-2.xml

```diff
@@ -42,15 +42,15 @@
 		<!-- lower robot -->
 		<include file="$(find cob_bringup)/drivers/sick_s300.launch" >
 			<arg name="robot" value="$(arg robot)" />
-			<arg name="name" value="front" />
+			<arg name="name" value="base_laser_front" />
```

Comment:
I chose this name as it is analog to the link name which is base_laser_X_link....also, I think a none-slash'ed __name__ is more convenient than a slash'ed name...